src/pages/index.js

```diff
@@ -75,12 +75,15 @@ function handleLikes(card) {
   api
     .likeCard(card.getId(), card.isLiked())
     .then((res) => {
-      card.toggleLikedStatus(res.isLiked);
+      card.toggleLikedStatus(card.isLiked);
+      card.handleLikesIcon();
+      return api.likeCard(card.getId(), isLiked), res.isLiked;
     })
```

Comment:
* When i like cards there is an error https://disk.yandex.ru/i/bG12aweDSP_T-Q 


Please, revise lines 79 and 80. You don't need them